spec/features/guest/sign_in_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+feature "Sign In" do
+  let(:user) { create :user, email: email, password: password }
```

Comment:
we can move it to before block because right now user cannot be created. also, you can read about `.let` and `.let!` https://relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let

Reply:
```ruby
before do
  create :user, email: email, password: password
end
```